src/test/suite/utils/preReleaseInstaller.utils.ts

```diff
@@ -79,19 +87,23 @@ export async function runInstallation(
   appName.value(options?.isInsidersApp || false);
   version.value(vscodeVersion);
   betaChannelEnabled.value(options?.isBetaChannelEnabled || false);
+  proposedAlphaChannelEnabled.value(
+    options?.isProposedAlphaChannelEnabled || false
+  );
   installedVersion.value(installed);
   const artifactUrl = getArtifactUrl(available);
-
+  console.log(artifactUrl);
   mockHttp(
     [[{ assets: [{ browser_download_url: artifactUrl }] }], LATEST_RELEASE_URL],
-    [{ data: "test" }, artifactUrl]
+    [{ data: "test" }, artifactUrl],
+    [{ data: "test" }, PROPOSED_ALPHA_ARTIFACTS_URL]
   );
 
   mockTempFile();
 
   return handlePreReleaseChannels(
     getContext({
-      [ALPHA_VERSION_KEY]: vscodeVersion,
+      [ALPHA_VERSION_KEY]: installed,
```

Comment:
this was wrong in my opinion, i changed it and all tests are still passing

Reply:
good catch